docs/internals/security.rst

```diff
@@ -37,6 +37,9 @@ Under these circumstances Borg guarantees that the attacker cannot
 The attacker can always impose a denial of service per definition (he could
 forbid connections to the repository, or delete it entirely).
 
+In an attack model where multiple clients independently access the same repository,
+Borg fails to provide confidentiality (i.e. guarantees 3) and 4) are essentially void).
```

Comment:
In an extended attack model (same as above, but with multiple clients independently updating the same repository), ...